pkg/daemon/ceph/osd/volume.go

```diff
@@ -455,6 +459,11 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		baseArgs = append(baseArgs, encryptedFlag)
 	}
 
+	// Because a breaking change, it is required to pass the --no-auto flag when running the "batch" command
+	if a.clusterInfo.CephVersion.IsAtLeast(cephVolumeLVMDiskSortingCephVersion) && !a.clusterInfo.CephVersion.IsOctopus() {
```

Comment:
With v15.2.6 will we just get a warning printed if we don't pass `--no-auto`, but the OSD will still work, right?